interface/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
```

```suggestion
    "noImplicitAny": true,
```
Or does this break things?